<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diagSelectWord</class>
 <widget class="QDialog" name="diagSelectWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Nihongo Companion</string>
  </property>
  <widget class="QLabel" name="labelField_in">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field to search on dictionary:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbField_in">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bSearch">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bConfirm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listResults">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bSkip">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
